src/main/java/com/gluonhq/substrate/target/AndroidTargetConfiguration.java

```diff
@@ -682,4 +685,32 @@ public BuildToolNotFoundException() {
             super("Android build tools not found. Please install it and try again.");
         }
     }
+
+    /**
+     * Scans the classpath and adds permission to AndroidManifest.xml for
+     * each attach service found.
+     * @param genManifest Path to the AndroidManifest.xml file
+     */
+    private void addRequiredPermissions(Path genManifest) {
```

Comment:
We had Glisten/Attach logic in Substrate in the past, and while that worked fine, we moved towards removing any external dependency, by using META_INF/Substrate as a way to allow any third party (JavaFX/Glisten/Attach...) pass configuration files to Substrate.

While this can be a complex task, I'd say we move this to each Attach service, using an xml file with the content that should be added to the final AndroidManifest.xml/Info.plist. 
Like:
```
<?xml version="1.0" encoding="utf-8"?>
<permissions>
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION"/>
</permissions>
```
This will be more flexible to include other nodes, like extra activities or services.

Then, in Substrate, we will scan these files, and combine them properly to produce the manifest/plist.

Reply:
+1 on the approach, but probably easier/more consistent if we use .json files (as we do with reflection/jni files)

Reply:
@jperedadnr  Good suggestion!